### ISSUE TITLE
How to test the model?

### ISSUE THREAD

**Issue by Chen65010445** [safakkbilici/Classification-On-Chest-X-Ray-Diagnosis-Images#1]
If I want to use it training other images, how to test the model I trained?

**Comment by safakkbilici** [comment]
Hello,

Code was specified for x-ray images; however, you can use it for other
dataser with a small modification. You can add your data folders
(train/test/val) in lines 89-94, and lines 101-122 is a plotting for x-ray
images that means you can delete this lines easily. Then the code
automatically tests the model on validation set in lines 226-240. The the
model's performance history automatically plotted in lines 304-323.

Sincerely,
*Şafak Bilici*

**Comment by Chen65010445** [comment]
Thank you. Because I already splitted my dataset into three parts : train, val and test. In training part I just want to get a best model. In testing part I want to use the test dataset to test the model and get a confusion-matrix. So is there good advices? I'm new in this area so maybe my question is so silly.

**Comment by safakkbilici** [comment]
All problems are welcome,

You could use the scikit-learn implementation. Scikit-learn has method for computing confusion matrix, for supporting numpy array instances. 
And torch tensors can be returned into numpy array.
Just get your predictions and targets using .numpy().
